pkg/operator/v410_00_assets/bindata.go

```diff
@@ -113,39 +116,108 @@ admission:
 apiServerArguments:
-  storage-backend:
-  - etcd3
-  storage-media-type:
-  - application/vnd.kubernetes.protobuf
+  allow-privileged:
+    - "true"
+  anonymous-auth:
+    - "true"
+  authorization-mode:
+    - Scope
+    - SystemMasters
+    - RBAC
+    - Node
+  audit-log-format:
+    - json
+  audit-log-maxbackup:
+    - "10"
+  audit-log-maxsize:
+    - "100"
+  audit-log-path:
+    - /var/log/kube-apiserver/audit.log
+  audit-policy-file: # this matches where .auditConfig.policyConfiguration is currently written.  We should update this.
+    - openshift.local.audit/policy.yaml
+  client-ca-file:
+    - /etc/kubernetes/static-pod-certs/configmaps/client-ca/ca-bundle.crt
   # switch to direct pod IP routing for aggregated apiservers to avoid service IPs as on source of instability
   enable-aggregator-routing:
-  - "true"
-  shutdown-delay-duration:
-  - 70s # give SDN some time to converge: 30s for iptable lock contention, 25s for the second try and some seconds for AWS to update ELBs
+    - "true"
+  enable-logs-handler:
+    - "false"
+  enable-swagger-ui:
+    - "true"
+  endpoint-reconciler-type:
+    - "lease"
+  etcd-cafile:
+    - /etc/kubernetes/static-pod-resources/configmaps/etcd-serving-ca/ca-bundle.crt
+  etcd-certfile:
+    - /etc/kubernetes/static-pod-resources/secrets/etcd-client/tls.crt
+  etcd-keyfile:
+    - /etc/kubernetes/static-pod-resources/secrets/etcd-client/tls.key
+  etcd-prefix:
+    - kubernetes.io
+  event-ttl:
+    - 3h
   http2-max-streams-per-connection:
-  - "2000"  # recommended is 1000, but we need to mitigate https://github.com/kubernetes/kubernetes/issues/74412
+    - "2000"  # recommended is 1000, but we need to mitigate https://github.com/kubernetes/kubernetes/issues/74412
+  insecure-port:
+    - "0"
+  kubelet-certificate-authority:
+    - /etc/kubernetes/static-pod-resources/configmaps/kubelet-serving-ca/ca-bundle.crt
+  kubelet-client-certificate:
+    - /etc/kubernetes/static-pod-resources/secrets/kubelet-client/tls.crt
+  kubelet-client-key:
+    - /etc/kubernetes/static-pod-resources/secrets/kubelet-client/tls.key
+  kubelet-https:
+    - "true"
   kubelet-preferred-address-types:
-  - InternalIP # all of our kubelets have internal IPs and we *only* support communicating with them via that internal IP so that NO_PROXY always works and is lightweight
-  # value needed to be bumped for scale tests.  The kube-apiserver did ok here
-  max-requests-inflight:
-  - "3000"
+    - InternalIP # all of our kubelets have internal IPs and we *only* support communicating with them via that internal IP so that NO_PROXY always works and is lightweight
+  kubelet-read-only-port:
+    - "0"
+  kubernetes-service-node-port:
+    - "0"
   # value should logically scale with max-requests-inflight
   max-mutating-requests-inflight:
-  - "1000"
+    - "1000"
+  # value needed to be bumped for scale tests.  The kube-apiserver did ok here
+  max-requests-inflight:
+    - "3000"
```

Comment:
previously it was set to `1200`

Reply:
it lied.  it was hard coded in origin.  it should be at least 3k